tests/dummyvpn/CMakeLists.txt

```diff
@@ -34,9 +34,17 @@ target_link_libraries(dummyvpn PRIVATE
 target_sources(dummyvpn PRIVATE
     ${CMAKE_SOURCE_DIR}/src/apps/vpn/platforms/dummy/dummycontroller.cpp
     ${CMAKE_SOURCE_DIR}/src/apps/vpn/platforms/dummy/dummycontroller.h
-    ${CMAKE_SOURCE_DIR}/src/apps/vpn/platforms/dummy/dummycryptosettings.cpp
 )
 
+if(${CMAKE_SYSTEM_NAME} STREQUAL "Linux")
```

Comment:
before this PR, the dummyvpn client does not include linux-specific code. But now, the "shared-component" is independent from the targets. Because of this, for linux, we need dbus.